index.html

```diff
@@ -48,6 +48,28 @@
 </head>
 
 <body>
+  <!-- <div> <input id=annoying-warning type=checkbox title="hide note">  -->
+    <div class=annoying-warning> 
+      <p><strong>Note:</strong> 
+      This specification met an urgent need at the time of publication,
+      but had significant drawbacks
+      particularly in fallback behavior, and should not be used for new content.</p>
+      <p>
+      New features in <a href="https://www.w3.org/TR/png-3/">PNG, Third Edition</a>
+      enable identification of BT.2100 PQ content 
+      using Coding Independent Code Points,
+      which is more compatible with established industry practice.
+      The <a href="https://www.w3.org/TR/png-3/#cICP-chunk">`cICP` chunk</a>
+      with values 9 16 0 1 serves the same purpose, with only 4 bytes of overhead.
+      </p>
+      <p>
+      In addition, the <a href="https://www.w3.org/TR/png-3/#mDCv-chunk">Mastering 
+      Display Color Volume</a> and
+      <a href="https://www.w3.org/TR/png-3/#cLLi-chunk">Content Light Level Information</a>
+      can be provided, for HRD10 compatibility.
+      </p>
+    </div>
```

Comment:
I suggest being less judgmental and more to the point:

This specification is deprecated and should not be used for new applications.

Applications are instead encouraged to consider using the [Third Edition of the PNG Recommendation](https://www.w3.org/TR/png-3/), which adds explicit support for HDR imagery, including images encoded according to Rec. ITU-R BT.2100. Signaling of HDR images is achieved using the [cICP chunk](https://www.w3.org/TR/png-3/#cICP-chunk), which uses the common Coding Independent Code Points specified in Rec. ITU-T H.273. For example, images that use the BT.2100 PQ color encoding are signaled with the cICP chunk components values `9 16 0 1`. In addition, the [Mastering Display Color Volume](https://www.w3.org/TR/png-3/#mDCv-chunk) and [Content Light Level Information](https://www.w3.org/TR/png-3/#cLLi-chunk) can be provided.

Reply:
I agree, and prefer this suggested phrasing.

Reply:
I pushed the suggested changes.

Except s/Recommendation/specification because it is not yet a Rec, but will soon be CR at which point we do want people to use it.